config/config.go

```diff
@@ -1243,14 +1241,6 @@ func (c *Config) addInput(name string, table *ast.Table) error {
 		return nil
 	}
 
-	// Legacy support renaming io input to diskio
-	if name == "io" {
-		if err := c.printUserDeprecation("inputs", name, nil); err != nil {
```

Comment:
It looks like this is removing a deprecation message for io. I think we want to do that still. Is the message printed somewhere else or do we need to add this back in?

Reply:
I agree, we should keep this deprecation notice.

Reply:
io is already in the deprecated plugins list, so a notice is still given..

Reply:
That's true. @reimda what do you think?

Reply:
I missed that io was in deprecations.go, thanks for pointing it out. That works for me.